### ISSUE TITLE
archinstall unusable 15min shut down wpa_supplicant

### ISSUE THREAD

**Issue by JeremiahCheatham** [archlinux/archinstall#1298]
I just tried the june 1 2022 archlinux "archinstall" for first time. I have been using my own archlinux install for a long time. HP chromebook 14 with 2995u. After installing initially it seemed to work but it wouldn't wake from sleep and then it wouldn't boot or take a very long time to book then it would freeze or crash. Shutdown was taking 15 minutes. I originally suspected btrfs since this was my first time using it. But it seems to be something to do with wpa_supplicant and networkmanager.

I chose the default option of taking the whole drive. btrfs compressed, kde and i added firefox that was it.

Since i had never used btrfs and also with compressions i assumed that was the problem so i spent hours searching that route. I was getting failed to unmount /home. However i saw it was hanging on this wpa_supplicant and networkmanager so i searched that. And wpa_supplicant seems to have huge google history of this problem. I am not sure if it's because iwd is also installed. 

/etc/NetworkManager/conf.d/wifi_backend.conf
````
[device]
wifi.backend=iwd
````

I added this and rebooted. And all was fixed it smoothly boots up and down and wakes up and all runs well.

How can wpa_supplicant completely cripple the system like this? Also this was a fresh install. So out of the box there is a major conflict happening that is rendering the system useless. I'm not sure if this is a bug in june 1 2022 or if it's been. but could someone shed some light on this. Like if this is a config problem why archinstall doing this way?

Side note firefox seems to be out of the box completely working with vaapi so there is nothing needed at all. However it would be nice if wayland switch was implemented somewhere. So firefox uses wayland not xwayland

````
if [ "$XDG_SESSION_TYPE" == "wayland" ]; then
    export MOZ_ENABLE_WAYLAND=1
fi
````

**Comment by JeremiahCheatham** [comment]
````
Hardware model detected: GOOGLE Falco; UEFI mode: True
Processor model detected: Intel(R) Celeron(R) 2955U @ 1.40GHz
Memory statistics: 3500776 available out of 3891596 total installed
Could not detect virtual system: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [256]: b'none\r\n'
System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [256]: b'none\r\n'
Virtualization detected: None; is VM: None
Graphics devices detected: dict_keys(['Intel Corporation Haswell-ULT Integrated Graphics Controller (rev 0b)'])
Disk states before installing: {'blockdevices': [{'name': 'loop0', 'fstype': 'squashfs', 'fsver': '4.0', 'label': None, 'uuid': None, 'fsavail': '0', 'fsuse%': '100%', 'mountpoints': ['/run/archiso/airootfs'], 'type': 'loop', 'size': '688.2M'}, {'name': 'sda', 'fstype': None, 'fsver': None, 'label': None, 'uuid': None, 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'disk', 'size': '14.9G', 'children': [{'name': 'sda1', 'fstype': 'vfat', 'fsver': 'FAT32', 'label': None, 'uuid': 'DF94-1D20', 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'part', 'size': '512M'}, {'name': 'sda2', 'fstype': 'ext4', 'fsver': '1.0', 'label': None, 'uuid': 'e08d39be-50f4-4b1b-8f93-b971cc369041', 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'part', 'size': '14.4G'}]}, {'name': 'sdb', 'fstype': 'iso9660', 'fsver': 'Joliet Extension', 'label': 'ARCH_202206', 'uuid': '2022-06-01-15-35-22-00', 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'disk', 'size': '7.2G', 'children': [{'name': 'sdb1', 'fstype': 'iso9660', 'fsver': 'Joliet Extension', 'label': 'ARCH_202206', 'uuid': '2022-06-01-15-35-22-00', 'fsavail': '0', 'fsuse%': '100%', 'mountpoints': ['/run/archiso/bootmnt'], 'type': 'part', 'size': '775M'}, {'name': 'sdb2', 'fstype': 'vfat', 'fsver': 'FAT16', 'label': 'ARCHISO_EFI', 'uuid': '785A-FB3F', 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'part', 'size': '86M'}]}]}
Testing connectivity to the Arch Linux mirrors ...
Hardware model detected: GOOGLE Falco; UEFI mode: True
Processor model detected: Intel(R) Celeron(R) 2955U @ 1.40GHz
Memory statistics: 3470208 available out of 3891596 total installed
Could not detect virtual system: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [256]: b'none\r\n'
System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [256]: b'none\r\n'
Virtualization detected: None; is VM: None
Graphics devices detected: dict_keys(['Intel Corporation Haswell-ULT Integrated Graphics Controller (rev 0b)'])
Disk states before installing: {'blockdevices': [{'name': 'loop0', 'fstype': 'squashfs', 'fsver': '4.0', 'label': None, 'uuid': None, 'fsavail': '0', 'fsuse%': '100%', 'mountpoints': ['/run/archiso/airootfs'], 'type': 'loop', 'size': '688.2M'}, {'name': 'sda', 'fstype': None, 'fsver': None, 'label': None, 'uuid': None, 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'disk', 'size': '14.9G', 'children': [{'name': 'sda1', 'fstype': 'vfat', 'fsver': 'FAT32', 'label': None, 'uuid': 'DF94-1D20', 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'part', 'size': '512M'}, {'name': 'sda2', 'fstype': 'ext4', 'fsver': '1.0', 'label': None, 'uuid': 'e08d39be-50f4-4b1b-8f93-b971cc369041', 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'part', 'size': '14.4G'}]}, {'name': 'sdb', 'fstype': 'iso9660', 'fsver': 'Joliet Extension', 'label': 'ARCH_202206', 'uuid': '2022-06-01-15-35-22-00', 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'disk', 'size': '7.2G', 'children': [{'name': 'sdb1', 'fstype': 'iso9660', 'fsver': 'Joliet Extension', 'label': 'ARCH_202206', 'uuid': '2022-06-01-15-35-22-00', 'fsavail': '0', 'fsuse%': '100%', 'mountpoints': ['/run/archiso/bootmnt'], 'type': 'part', 'size': '775M'}, {'name': 'sdb2', 'fstype': 'vfat', 'fsver': 'FAT16', 'label': 'ARCHISO_EFI', 'uuid': '785A-FB3F', 'fsavail': None, 'fsuse%': None, 'mountpoints': [None], 'type': 'part', 'size': '86M'}]}]}
Testing connectivity to the Arch Linux mirrors ...
 -- Chosen configuration --
{
    "HSM": null,
    "__separator__": null,
    "additional-repositories": [],
    "archinstall-language": "English",
    "audio": "pipewire",
    "bootloader": "systemd-bootctl",
    "config_version": "2.5.0",
    "debug": false,
    "desktop-environment": "kde",
    "gfx_driver": "Intel (open-source)",
    "harddrives": [
        "/dev/sda"
    ],
    "hostname": "Arch",
    "kernels": [
        "linux"
    ],
    "keyboard-layout": "uk",
    "mirror-region": {
        "United Kingdom": {
            "http://archlinux.uk.mirror.allworldit.com/archlinux/$repo/os/$arch": true,
            "http://lon.mirror.rackspace.com/archlinux/$repo/os/$arch": true,
            "http://mirror.bytemark.co.uk/archlinux/$repo/os/$arch": true,
            "http://mirror.cov.ukservers.com/archlinux/$repo/os/$arch": true,
            "http://mirror.netweaver.uk/archlinux/$repo/os/$arch": true,
            "http://mirrors.manchester.m247.com/arch-linux/$repo/os/$arch": true,
            "http://mirrors.melbourne.co.uk/archlinux/$repo/os/$arch": true,
            "http://www.mirrorservice.org/sites/ftp.archlinux.org/$repo/os/$arch": true,
            "https://archlinux.uk.mirror.allworldit.com/archlinux/$repo/os/$arch": true,
            "https://lon.mirror.rackspace.com/archlinux/$repo/os/$arch": true,
            "https://mirror.bytemark.co.uk/archlinux/$repo/os/$arch": true,
            "https://mirror.cov.ukservers.com/archlinux/$repo/os/$arch": true,
            "https://mirror.netweaver.uk/archlinux/$repo/os/$arch": true,
            "https://mirrors.melbourne.co.uk/archlinux/$repo/os/$arch": true,
            "https://www.mirrorservice.org/sites/ftp.archlinux.org/$repo/os/$arch": true
        }
    },
    "mount_point": null,
    "nic": {
        "dhcp": true,
        "dns": null,
        "gateway": null,
        "iface": null,
        "ip": null,
        "type": "nm"
    },
    "ntp": true,
    "packages": [
        "firefox"
    ],
    "plugin": null,
    "profile": {
        "path": "/usr/lib/python3.10/site-packages/archinstall/profiles/desktop.py"
    },
    "save_config": null,
    "script": "guided",
    "silent": false,
    "swap": true,
    "sys-encoding": "UTF-8",
    "sys-language": "en_US",
    "timezone": "Europe/London",
    "version": "2.5.0"
}
{
    "/dev/sda": {
        "partitions": [
            {
                "boot": true,
                "encrypted": false,
                "filesystem": {
                    "format": "fat32"
                },
                "mountpoint": "/boot",
                "size": "512MiB",
                "start": "1MiB",
                "type": "primary",
                "wipe": true
            },
            {
                "btrfs": {
                    "subvolumes": {
                        "@": "/",
                        "@.snapshots": "/.snapshots",
                        "@home": "/home",
                        "@log": "/var/log",
                        "@pkg": "/var/cache/pacman/pkg"
                    }
                },
                "encrypted": false,
                "filesystem": {
                    "format": "btrfs",
                    "mount_options": [
                        "compress=zstd"
                    ]
                },
                "mountpoint": null,
                "size": "100%",
                "start": "513MiB",
                "type": "primary",
                "wipe": true
            }
        ],
        "wipe": true
    }
}
Creating a new partition label on /dev/sda
Adding partition to BlockDevice(/dev/sda, size=14.9GB, free_space=1031kB, bus_type=sata), 1MiB->512MiB
Adding partition using the following parted command: /dev/sda mkpart primary fat32 1MiB 512MiB
Formatting /dev/sda1 -> vfat
Could not locate mount information for /dev/sda1: ['/usr/bin/findmnt', '--json', '-R', '/dev/sda1'] exited with abnormal exit code [256]: b''
Marking partition Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat) as bootable.
Setting boot on on (parted) partition index 1
Adding partition to BlockDevice(/dev/sda, size=14.9GB, free_space=1031kB, bus_type=sata), 513MiB->100%
Adding partition using the following parted command: /dev/sda mkpart primary btrfs 513MiB 100%
Formatting /dev/sda2 -> btrfs
Could not locate mount information for /dev/sda2: ['/usr/bin/findmnt', '--json', '-R', '/dev/sda2'] exited with abnormal exit code [256]: b''
Could not locate mount information for /dev/sda2: ['/usr/bin/findmnt', '--json', '-R', '/dev/sda2'] exited with abnormal exit code [256]: b''
Mounting BTRFSPartition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs) to /mnt/archinstall/
Setting up subvolumes: {'@': '/', '@home': '/home', '@log': '/var/log', '@pkg': '/var/cache/pacman/pkg', '@.snapshots': '/.snapshots'}
Attempting to create subvolume at /mnt/archinstall/@
Attempting to create subvolume at /mnt/archinstall/@home
Attempting to create subvolume at /mnt/archinstall/@log
Attempting to create subvolume at /mnt/archinstall/@pkg
Attempting to create subvolume at /mnt/archinstall/@.snapshots
Could not locate mount information for /dev/sda1: ['/usr/bin/findmnt', '--json', '-R', '/dev/sda1'] exited with abnormal exit code [256]: b''
Mounting /boot to /mnt/archinstall/boot using Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat)
Using mount order: [('/', <function Installer.mount_ordered_layout.<locals>.<lambda> at 0x7f5cf4b196c0>), ('/.snapshots', <function Installer.mount_ordered_layout.<locals>.<lambda> at 0x7f5cf4b18160>), ('/boot', <function Installer.mount_ordered_layout.<locals>.<lambda> at 0x7f5cf4b18a60>), ('/home', <function Installer.mount_ordered_layout.<locals>.<lambda> at 0x7f5cf4b18940>), ('/var/cache/pacman/pkg', <function Installer.mount_ordered_layout.<locals>.<lambda> at 0x7f5cf4b18040>), ('/var/log', <function Installer.mount_ordered_layout.<locals>.<lambda> at 0x7f5cf4b18430>)]
Could not locate mount information for /dev/sda2: ['/usr/bin/findmnt', '--json', '-R', '/dev/sda2'] exited with abnormal exit code [256]: b''
Mounting subvolume @ on BTRFSPartition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs) to /mnt/archinstall
Getting mount information for device path /mnt/archinstall
Mounting subvolume @.snapshots on BTRFSPartition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall) to /mnt/archinstall/.snapshots
Getting mount information for device path /mnt/archinstall/.snapshots
Could not locate mount information for /dev/sda1: ['/usr/bin/findmnt', '--json', '-R', '/dev/sda1'] exited with abnormal exit code [256]: b''
Could not locate mount information for /dev/sda1: ['/usr/bin/findmnt', '--json', '-R', '/dev/sda1'] exited with abnormal exit code [256]: b''
Mounting Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat) to /mnt/archinstall/boot
Getting mount information for device path /mnt/archinstall/boot
Mounting subvolume @home on BTRFSPartition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall) to /mnt/archinstall/home
Getting mount information for device path /mnt/archinstall/home
Mounting subvolume @pkg on BTRFSPartition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall) to /mnt/archinstall/var/cache/pacman/pkg
Getting mount information for device path /mnt/archinstall/var/cache/pacman/pkg
Mounting subvolume @log on BTRFSPartition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall) to /mnt/archinstall/var/log
Getting mount information for device path /mnt/archinstall/var/log
Unreliable results might be given for /dev/sdb1 due to partprobe error: ['/usr/bin/partprobe', '/dev/sdb'] exited with abnormal exit code [256]: b'Error: Partition(s) 1 on /dev/sdb have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.\r\n'
Unreliable results might be given for /dev/sdb1 due to partprobe error: ['/usr/bin/partprobe', '/dev/sdb'] exited with abnormal exit code [256]: b'Error: Partition(s) 1 on /dev/sdb have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.\r\n'
Filtering available mounts {'/mnt/archinstall': Partition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall), '/run/archiso/bootmnt': Partition(path=/dev/sdb1, size=0.8, PARTUUID=None, fs=iso9660, mounted=/run/archiso/bootmnt), '/mnt/archinstall/boot': Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat, mounted=/mnt/archinstall/boot)} to those under /mnt/archinstall
Available partitions: {'/mnt/archinstall': Partition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall), '/mnt/archinstall/boot': Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat, mounted=/mnt/archinstall/boot)}
Waiting for automatic mirror selection (reflector) to complete.
Waiting for dbus-org.freedesktop.timesync1.service to enter running state
A new package mirror-list has been created: /etc/pacman.d/mirrorlist
Unreliable results might be given for /dev/sdb1 due to partprobe error: ['/usr/bin/partprobe', '/dev/sdb'] exited with abnormal exit code [256]: b'Error: Partition(s) 1 on /dev/sdb have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.\r\n'
Unreliable results might be given for /dev/sdb1 due to partprobe error: ['/usr/bin/partprobe', '/dev/sdb'] exited with abnormal exit code [256]: b'Error: Partition(s) 1 on /dev/sdb have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.\r\n'
Filtering available mounts {'/mnt/archinstall': Partition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall), '/run/archiso/bootmnt': Partition(path=/dev/sdb1, size=0.8, PARTUUID=None, fs=iso9660, mounted=/run/archiso/bootmnt), '/mnt/archinstall/boot': Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat, mounted=/mnt/archinstall/boot)} to those under /mnt/archinstall
Available partitions: {'/mnt/archinstall': Partition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall), '/mnt/archinstall/boot': Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat, mounted=/mnt/archinstall/boot)}
System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [256]: b'none\r\n'
The testing flag is not set. This system will be installed without testing repositories enabled.
The testing flag is not set. This system will be installed without testing repositories enabled.
Installing packages: ['base', 'base-devel', 'linux-firmware', 'linux', 'btrfs-progs', 'intel-ucode']
Enabling periodic TRIM
Enabling service fstrim.timer
Setting up swap on zram
Installing packages: ('zram-generator',)
Enabling service systemd-zram-setup@zram0.service
Unreliable results might be given for /dev/sdb1 due to partprobe error: ['/usr/bin/partprobe', '/dev/sdb'] exited with abnormal exit code [256]: b'Error: Partition(s) 1 on /dev/sdb have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.\r\n'
Unreliable results might be given for /dev/sdb1 due to partprobe error: ['/usr/bin/partprobe', '/dev/sdb'] exited with abnormal exit code [256]: b'Error: Partition(s) 1 on /dev/sdb have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.\r\n'
Filtering available mounts {'/mnt/archinstall': Partition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall), '/run/archiso/bootmnt': Partition(path=/dev/sdb1, size=0.8, PARTUUID=None, fs=iso9660, mounted=/run/archiso/bootmnt), '/mnt/archinstall/boot': Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat, mounted=/mnt/archinstall/boot)} to those under /mnt/archinstall
Available partitions: {'/mnt/archinstall': Partition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall), '/mnt/archinstall/boot': Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat, mounted=/mnt/archinstall/boot)}
Adding bootloader systemd-bootctl to Partition(path=/dev/sda1, size=0.5, PARTUUID=11D3-400C, fs=vfat, mounted=/mnt/archinstall/boot)
Installing packages: ('efibootmgr',)
System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [256]: b'none\r\n'
Getting mount information for device path /mnt/archinstall
Identifying root partition by PARTUUID on Partition(path=/dev/sda2, size=14.4, PARTUUID=3cdb126c-f812-4579-a36c-ec5f4095b411, fs=btrfs, mounted=/mnt/archinstall), looking for '9a3a7245-2367-4f47-a9d5-990615fe0444'.
Installing packages: ['networkmanager']
Installing packages: ['network-manager-applet']
Enabling service NetworkManager.service
This audio server will be used: pipewire
Installing pipewire
Installing packages: ['pipewire', 'pipewire-alsa', 'pipewire-jack', 'pipewire-pulse', 'gst-plugin-pipewire', 'libpulse', 'wireplumber']
Installing packages: ['firefox']
Installing archinstall profile Profile(desktop)
Installing packages: ['nano', 'vim', 'openssh', 'htop', 'wget', 'iwd', 'wireless_tools', 'wpa_supplicant', 'smartmontools', 'xdg-utils']
Installing archinstall profile Profile(kde)
Installing archinstall profile Profile(xorg)
Installing packages: ('xorg-server xorg-xinit mesa libva-intel-driver intel-media-driver vulkan-intel',)
Installing packages: ['plasma-meta', 'konsole', 'kwrite', 'dolphin', 'ark', 'sddm', 'plasma-wayland-session', 'egl-wayland']
Enabling service sddm
Creating user jeremiah
Setting password for jeremiah
Enabling sudo permissions for jeremiah.
Activating systemd-timesyncd for time synchronization using Arch Linux and ntp.org NTP servers.
Enabling service systemd-timesyncd
Setting password for root
Setting keyboard language to uk
Keyboard language for this installation is now set to: uk
Updating /mnt/archinstall/etc/fstab
For post-installation tips, see https://wiki.archlinux.org/index.php/Installation_guide#Post-installation
Installation completed without any errors. You may now reboot.

````

**Comment by svartkanin** [comment]
I think this should be no longer an issue if so we can reopen it
